src/figuresClasses.ts

```diff
@@ -1,11 +1,101 @@
-export interface Figure {}
+export interface Figure {
+  shape: string;
+  color: string;
+  getArea(): number;
+}
+
+function triangleArea(a: number, b: number, c: number): number {
+  const s = (a + b + c) / 2;
+
+  return Math.sqrt(s * (s - a) * (s - b) * (s - c));
+}
+
+function circleArea(radius: number): number {
+  return radius ** 2 * Math.PI;
+}
+
+function rectangleArea(width: number, height: number): number {
+  return width * height;
+}
+
+export class Triangle implements Figure {
+  constructor(
+    public color: string,
+    public a: number,
+    public b: number,
+    public c: number,
+    public shape = 'triangle',
+  ) {
+    this.getArea();
+  }
+
+  getArea(): number {
+    const sides = [this.a, this.b, this.c];
+    const area = triangleArea(this.a, this.b, this.c);
+    const indexOfLongest = sides.indexOf(Math.max(...sides));
+    let counter = 0;
+
+    if (Math.min(...sides) <= 0) {
+      throw new Error();
```

Comment:
[CODE STYLE] - When throwing an error, try to provide a detailed description in the error message. In this case, the error message could be 'Triangle side length should be greater than 0'.